File: client_manage_web/src/main/webapp/js/service/ordersService.js
//服务层
app.service('ordersService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../orders/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../orders/findPage.do?pageNum='+page+'&pageSize='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../orders/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../orders/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../orders/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../orders/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../orders/search.do?pageNum='+page+"&pageSize="+rows, searchEntity);
	}    	
});

File: cinema_client_interface/src/main/java/cn/edu/aynu/rjxy/client/service/OrdersService.java
package cn.edu.aynu.rjxy.client.service;


import cn.edu.aynu.rjxy.pojo.TbOrders;
import com.github.pagehelper.PageInfo;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface OrdersService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbOrders> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageInfo findPage(int pageNum, int pageSize);


	/**
	 * 增加
	*/
	public void add(TbOrders orders);


	/**
	 * 修改
	 */
	public void update(TbOrders orders);


	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbOrders findOne(String orderId);


	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(String[] orderIds);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageInfo findPage(TbOrders orders, int pageNum, int pageSize);
	
}
